ui/src/typechain-types/@openzeppelin/contracts/access/Ownable.ts

```diff
@@ -0,0 +1,176 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
are you sure that you would like to put autogenerated files into git?

Reply:
discussed in DM, worth to improve